pkg/storage/caching/caching.go

```diff
@@ -16,6 +17,7 @@ var _ storage.OpenFGADatastore = (*cachedOpenFGADatastore)(nil)
 
 type cachedOpenFGADatastore struct {
```

Comment:
Do we not need to initialize the singleflight group in the constructor?

Reply:
No need. But if you prefer explicitness it can be done.